todo_app/deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+echo $DOCKER_PASSWORD | docker login --username $DOCKER_USER --password-stdin
+docker build --target production --cache-from kevineatondevops/todo-app:latest --tag kevineatondevops/todo-app:latest --tag kevineatondevops/todo-app:$TRAVIS_COMMIT --tag registry.heroku.com/kedevopstodoapp/web .
+docker push kevineatondevops/todo-app:latest
```

Comment:
Above you've tagged the container for both the `$TRAVIS_COMMIT` and `latest` but we only push the latest one - if you want the commit container to be available we should also push that